core/src/banking_stage.rs

```diff
@@ -442,21 +440,57 @@ impl BankingStage {
         let data_budget = Arc::new(DataBudget::default());
         let batch_limit =
             TOTAL_BUFFERED_PACKETS / ((num_threads - NUM_VOTE_PROCESSING_THREADS) as usize);
+        // Keeps track of extraneous vote transactions for the vote threads
+        let latest_unprocessed_votes = Arc::new(LatestUnprocessedVotes::new());
+        let should_split_voting_threads = bank_forks
+            .read()
+            .map(|bank_forks| {
+                let bank = bank_forks.root_bank();
+                bank.feature_set
+                    .is_active(&allow_votes_to_directly_update_vote_state::id())
```

Comment:
yeah originally we were considering a hot swap approach where we check root bank every loop to decide however this seemed cumbersome. Not to mention that this hot swap would then require another feature in order to activate and with the rate that feature activations seem to be going it would be quicker to just wait for a restart.